### PR TITLE
Adding logging framework to the gdeploy file

### PR DESCRIPTION
**Pull request by devyanikota**
@sac Can you please take a look at this and let me know if any more changes are needed

### CODE REVIEW

File: gdeploy/gdeploy [hunk 1]
```diff
@@ -135,41 +136,41 @@ def create_playbooks_in_local():
              % (playbooks_path_pkg)
         print "\nError: " + msg
         Global.logger.error(msg.strip('\n\t\r'))
-        if trace:
-            Global.logger.info("**** Either the path to playbooks is incorrect or env variable"\
-                               "GDEPLOY_TEMPLATES is not set properly ****")
+        if Global.trace:
+            Global.logger.info("Either the path to playbooks is incorrect or env variable"\
+                               "GDEPLOY_TEMPLATES is not set properly.")
```
Comment (sac) [inline gdeploy/gdeploy]: Add a space before GDEPLOY_TEMPLATES

File: gdeploy/gdeploy [hunk 2]
```diff
@@ -125,22 +136,41 @@ def create_playbooks_in_local():
              % (playbooks_path_pkg)
         print "\nError: " + msg
         Global.logger.error(msg.strip('\n\t\r'))
+        if Global.trace:
+            Global.logger.info("Either the path to playbooks is incorrect or env variable"\
+                               "GDEPLOY_TEMPLATES is not set properly.")
```
Comment (sac) [inline gdeploy/gdeploy]: Add a space before GDEPLOY_TEMPLATES

**Review by sac — CHANGES_REQUESTED** [review]
Devyani, can you please make that cosmetic change?

File: gdeploy/gdeploy (hunk 2 above)
Comment (devyanikota) [reply gdeploy/gdeploy]: @sac hi :) Rectified the cosmetic change, please review the same :) thanks.

**Pull request merged**